### ISSUE TITLE
SVG styles are being ignored

### ISSUE THREAD

**Issue by adityatj** [jakubpawlowicz/clean-css#1021]
Hello,

Firstly thank you for the awesome library.

My issue is with the svg styles being skipped/not appearing in the final output CSS.

I use a grunt plugin `grunt-contrib-cssmin`

### Environment

* grunt-contrib-cssmin version - 2.2.1
* clean-css version - 4.1.11
* node.js version - 6.11.3
* operating system: Windows 8

### Input CSS

```css
.node:hover circle {
    r: 18;
    stroke: orange;
}
```

### Actual output CSS

```css
.node:hover circle {
    stroke: orange;
}
```

### Expected output CSS

```css
.node:hover circle {
    r: 18;
    stroke: orange;
}
```

**Comment by Pedro-vk** [comment]
I've found this error in an Angular project. AngularCLI implements clean-css on production builds.
I've tested with units (like `r: 20px`) but it doesn't work fine.

**Comment by Dari4sho** [comment]
I'm encountering this as well. I've got this definition 
`@keyframes pulse-me{
  0%{
    r: 4.1;
    opacity: 0;
  }
  50%{
    opacity: 0.3;
  }
  70%{
    opacity: 0.1;
  }
  100%{
    r: 16;
    opacity: 0;
  }
}`

and it converts to 

`@keyframes pulse-me{0%{opacity:0}50%{opacity:.3}70%{opacity:.1}100%{opacity:0}}`

I tried a couple of options but I couldn't find the reason.

**Comment by prabhjyotsingh** [comment]
Is there any chance of this being fixed?

**Comment by prabhjyotsingh** [comment]
Is there any chance of this being fixed?
What it looks like is this regex (https://github.com/jakubpawlowicz/clean-css/blob/master/lib/optimizer/level-1/optimize.js#L35) is look for >= 3 char in the property name.

**Comment by hasmanyguitars** [comment]
I'm seeing a similar issue with cy/cx properties.

**Comment by jakubpawlowicz** [comment]
Fixed on main branch, will be released in clean-css 5.0 soon.
